man/DockerRepository.Rd

```diff
@@ -45,6 +45,8 @@ An object of type DockerRepository
 % You can use \dontrun{} for text that should only be shown, but not run, and \dontshow{} for extra commands 
 % for testing that should not be shown to users, but will be run by example(). 
 \examples{
+dr <- DockerRepository(repositoryName = "test", parent = "syn123")
```

Comment:
Include a reference to an explanation that this object is used to create a reference to a Docker repository in an *external* registry.  To push a docker repository to the *Synapse* registry, use the Docker client rather than the R client.

Reply:
We already address this:
> NOTE: store()-ing a DockerRepository created in the Python client will always result in it being treated as a reference to an external Docker repository that is not managed by synapse. 
To upload a docker image that is managed by Synapse please use the official Docker client and read http://docs.synapse.org/articles/docker.html for instructions on uploading a Docker Image to Synapse